logging_test.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+	"time"
+)
+
+type mockClock struct {
+	time time.Time
+}
+
+func (c *mockClock) now() time.Time {
+	return c.time
+}
+
+func TestRFCPrefixWriter(t *testing.T) {
```

Comment:
Idea: Use more than one line for test.
That would make the test more robust - just in case.